test/test_thingspeak_rate_limiting.py

```
@@ -1,252 +1,223 @@
 #!/usr/bin/env python3
 """
-Тест ограничения частоты запросов к ThingSpeak
-Проверяет логику защиты от спама и ограничения на 1 час при множественных ошибках
+Тест логики ограничения частоты запросов к ThingSpeak
+Проверяет алгоритм блокировки на 1 час при множественных ошибках
 """
 
 import sys
-import os
 import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)